packages/gitbook/src/components/DocumentView/InlineLink/InlineLink.tsx

```diff
@@ -105,6 +113,14 @@ function InlineLinkTooltipWrapper(props: {
             },
         ];
     }
+    if (isMailto) {
+        resolved.text = resolved.text.split('mailto:')[1] ?? resolved.text;
+        breadcrumbs = [
+            {
+                label: tString(language, 'link_tooltip_email'),
+            },
+        ];
+    }
```

Comment:
Same better to do:

```ts
if (isMailto) {
  ...
} else if (isExternal) {
  ...
}
```

It makes it clearer that the two options are exclusive. You could also do it for `isSamePage`.